### PR TITLE
Milestone 9: Add contact validation

### PR DESCRIPTION
**Pull request by yonas44**
- Add span to display error message.
- Add event listeners for the email input field.
- Add function to validate the input from the field.

Thankyou for reviewing my PR!

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -119,3 +124,38 @@ navAboutme.addEventListener('click', closeMenu);
...
+    if (event.data.toLowerCase() !== event.data) {
+      email.setCustomValidity('Please use LOWERCASE letters, Thanks!');
+      email.reportValidity();
+    } else if (email.validity.typeMismatch) {
+      email.setCustomValidity('please type a valid Email here!');
+      email.reportValidity();
+    } else {
+      email.setCustomValidity('');
+    }
+  } else if (event.data === null) footerSpan.style.display = 'none';
```
Comment (omar-muhamad) [inline script.js]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=If%20the%20validation%20is%20not%20OK%2C%20you%20show%20an%20error%20message%20to%20the%20user%20near%20the%20submit%20button%20informing%20them%20of%20the%20error%20and%20the%20form%20is%20not%20sent.), Please make the errpr message appear near the submit button.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: script.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply script.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @yonas44 ,

### Status: Approved ✔️ ✔️ ✔️

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏
## Optional Suggestion

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by yonas44** [comment]
Thanks for the review!

**Pull request merged**